nil/services/rpc/internal/http/connection.go

```diff
@@ -3,18 +3,20 @@ package http
 import (
 	"io"
 	"net/http"
+	"strconv"
 	"time"
 
+	"github.com/NilFoundation/nil/nil/common/check"
 	"github.com/NilFoundation/nil/nil/common/version"
 )
 
 const (
 	MaxRequestContentLength  = 1024 * 1024 * 32 // 32MB
-	currentHeight            = 356
-	minSupportedRevision     = version.BaseRevision + currentHeight
 	minSupportedNiljsVersion = "0.24.0"
 )
 
+var minSupportedRevision = 1
```

Comment:
Keep this as a constant. We do need to set this manually (likely)

Reply:
Done!